scripts/SimplexUI/loadUiType.py

```diff
@@ -21,39 +21,23 @@
 import xml.etree.ElementTree as xml
 from cStringIO import StringIO
 
-try:
-	from PySide import QtGui, QtCore
-	from PySide.QtCore import Signal, Slot
-	from PySide.QtCore import Qt, QObject, QTimer, QPoint, QEvent, QSettings, QModelIndex
-	from PySide.QtGui import QMessageBox, QInputDialog, QFileDialog, QMenu, QApplication, QSortFilterProxyModel, QAction
-	from PySide.QtGui import QDialog, QMainWindow, QSplashScreen, QShortcut, QItemSelection, QProgressDialog
-	from PySide.QtGui import QStandardItemModel, QStandardItem, QKeySequence, QCursor, QMouseEvent
-	import pysideuic as uic
-	def toPyObject(thing):
+def toPyObject(thing):
+	try:
+		return thing.toPyObject()
+	except:
 		return thing
 
+from Qt import QtGui, QtCore, QtWidgets, QtCompat
+from Qt.QtCore import Signal, QSortFilterProxyModel, Slot, QModelIndex
+from Qt.QtCore import Qt, QObject, QTimer, QPoint, QEvent, QItemSelection, QSettings
+from Qt.QtWidgets import QMessageBox, QInputDialog, QFileDialog, QMenu, QApplication, QAction
+from Qt.QtWidgets import QDialog, QMainWindow, QSplashScreen, QShortcut, QProgressDialog
+from Qt.QtGui import QStandardItemModel, QStandardItem, QKeySequence, QCursor, QMouseEvent
+try:
+	import pyside2uic as uic
 except ImportError:
-	try:
-		from PyQt4 import QtGui, QtCore
-		from PyQt4.QtCore import pyqtSignal as Signal, pyqtSlot as Slot
-		from PyQt4.QtCore import Qt, QObject, QTimer, QPoint, QEvent, QSettings, QModelIndex
-		from PyQt4.QtGui import QMessageBox, QInputDialog, QFileDialog, QMenu, QApplication, QSortFilterProxyModel, QAction
-		from PyQt4.QtGui import QDialog, QMainWindow, QSplashScreen, QShortcut, QItemSelection, QProgressDialog
-		from PyQt4.QtGui import QStandardItemModel, QStandardItem, QKeySequence, QCursor, QMouseEvent
-		import PyQt4.uic as uic
-		def toPyObject(thing):
-			return thing.toPyObject()
-
-	except ImportError:
-		from PySide2 import QtGui, QtCore
-		from PySide2.QtCore import Signal, QSortFilterProxyModel, Slot, QModelIndex
-		from PySide2.QtCore import Qt, QObject, QTimer, QPoint, QEvent, QItemSelection, QSettings
-		from PySide2.QtWidgets import QMessageBox, QInputDialog, QFileDialog, QMenu, QApplication, QAction
-		from PySide2.QtWidgets import QDialog, QMainWindow, QSplashScreen, QShortcut, QProgressDialog
-		from PySide2.QtGui import QStandardItemModel, QStandardItem, QKeySequence, QCursor, QMouseEvent
-		import pyside2uic as uic
-		def toPyObject(thing):
-			return thing
+	uic = QtCompat.loadUi
```

Comment:
Put the `QtCompat.loadUi` call into the `__init__`, and then pass `self` as the `baseinstance` argument.  This will load the ui file directly onto the class.  Like so:
```python
class SimplexDialog(QMainWindow):
	def __init__(self, parent=None, dispatch=None):
		super(SimplexDialog, self).__init__(parent)
		QtCompat.loadUi(utils.get_ui_file(__file__), self)
```